huaweicloud/services/dis/resource_huaweicloud_dis_stream.go

```diff
@@ -349,7 +351,7 @@ func checkPartitionUpdateResult(ctx context.Context, client *golangsdk.ServiceCl
 			if err != nil {
 				return nil, "failed", err
 			}
-			logp.Printf("WritablePartitionCount=", resp.WritablePartitionCount, targetValue)
+			log.Printf("WritablePartitionCount: %d, targetValue: %d", resp.WritablePartitionCount, targetValue)
```

Comment:
missing logging level

```suggestion
			log.Printf("[DEBUG] WritablePartitionCount: %d, targetValue: %d", resp.WritablePartitionCount, targetValue)
```